aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -840,7 +840,6 @@ def presubmit(self, folder: Folder) -> CalcInfo:
         job_tmpl = JobTemplate()
         job_tmpl.submit_as_hold = False
         job_tmpl.rerunnable = self.options.get('rerunnable', False)
-        job_tmpl.job_environment = {}
```

Comment:
No it shouldn't. It was a pointless initialization because it would later be overwritten in the same method:
https://github.com/aiidateam/aiida-core/blob/f5813092879a7204e5d4d51da463841bb32881eb/aiida/engine/processes/calcjobs/calcjob.py#L984